os/arch/arm/src/amebad/amebad_i2s.c

```diff
@@ -116,10 +117,13 @@
 
 #define OVER_SAMPLE_RATE (384U)
 
-#define I2S_DMA_PAGE_SIZE	768   // 2 ~ 4096
-#define I2S_DMA_PAGE_NUM    4   // Vaild number is 2~4
+#define I2S_DMA_PAGE_SIZE	1024   /* 4 ~ 16384, set to a factor of APB size */
+#define I2S_DMA_PAGE_NUM    4   /* Vaild number is 2~4 */
 
 /* I2S buffer container */
+#ifdef __16_BIT_PADDING__
+uint8_t audio_buf[I2S_DMA_PAGE_SIZE];
```

Comment:
If this is used only in this file internally, let's add `static`.

Reply:
Yes it is only used within amebad_i2s.c, have added static to this variable